brink.conf

```diff
@@ -1,7 +1,7 @@
-BASE_REQUIREMENTS='pip==20.1.1 chevah-brink==0.75 paver==1.2.4'
-PYTHON_CONFIGURATION='default@2.7.18.4b61bd67:lnx@2.7.18.99d5a68f'
-BINARY_DIST_URI='http://binary.chevah.com/production'
-PIP_INDEX='http://pypi.chevah.com/simple'
+BASE_REQUIREMENTS='pip==20.2.4 chevah-brink==0.78.1 paver==1.2.4'
+PYTHON_CONFIGURATION='default@2.7.18.bd461724'
```

Comment:
why is here still 2.7.18?

Reply:
> why is here still 2.7.18?

Theoretically speaking, if we still have to use Buildbot?

Reply:
True.. Thaks. I am working on migrating the buildbot tools to py3...so that we can be py3 only.